### DISCUSSION TITLE
Pares TV

### DISCUSSION THREAD

**Discussion by AntiPontifex** [iptv-org/community#356] [Show and tell]
### Channel ID

ParesTV.ar

### Stream URL

https://live.vimeocdn.com/1663881628-0x7b024d231f708803e3384adbcd1b716a2d93e396/6e287571-5a16-4a7b-a35d-f39e54b0a44b/chunklist_b728000.m3u8?max_segments=10

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Sphinxroot** [comment]
https://parestv.com.ar/

tokenized
https://live-ak.vimeocdn.com/exp=1663918325~acl= .............
https://live-api.vimeocdn.com/sessions/fda8dfd9-b8e5-43b3-87aa-254ec4ee7d2c/playlist/hls.m3u8?cdn=akamai&json=1&~id=player&~exp=1663916400&~sig= ..........


<html><body>


referrer | https://vimeo.com/event/1191635/embed?transparent=0&autoplay=1
-- | --


</body>
</html>
private setting

**Comment by AntiPontifex** [comment]
Help with this one, Lane?

**Comment by freearhey** [comment]
https://live.vimeocdn.com/1663881628-0x7b024d231f708803e3384adbcd1b716a2d93e396/6e287571-5a16-4a7b-a35d-f39e54b0a44b/chunklist_b728000.m3u8?max_segments=10 (offline)
<img width="833" alt="image" src="https://user-images.githubusercontent.com/7253922/232235387-a5865449-496f-4189-8140-a079e89c2ee5.png">
